UPLOAD/inc/languages/english/admin/config_purgesoftdeleted.lang.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$l['purgesoftdeleted_name'] = "Purge soft deleted posts and threads";
+$l['purgesoftdeleted_desc'] = "Automatically purges soft deleted posts and threads";
+
+$l['purgesoftdeleted_task_name'] = "Purge soft deleted posts and threads";
+$l['purgesoftdeleted_task_desc'] = "Checks for soft deleted posts and threads and purges them automatically.";
+
+$l['purgesoftdeleted_task_log'] = "Purge soft deleted posts and threads now are pernamenly deleted";
```

Comment:
Typo: "pernamenly" => "permanently". Also, we can probably strip the "Purge" from the start of this message, and replace it with an indication that age matters, for something like:

"Older soft deleted posts and threads were permanently deleted"

Reply:
Fixed, thx!